kiosk/src/BackendRequests.tsx

```diff
@@ -48,6 +55,7 @@ export const addGameToKioskAsync = async (kioskId: string | undefined, gameShare
         await response.json();
     }
     catch (error) {
+        //dispatch here
```

Comment:
This will be the error handling I do. I don't think I'll actually be doing dispatches, but rather just plain JavaScript alerts for the time being. I hope to change this in the future to something similar to what multiplayer does, but my understanding of React kept me from doing this for kiosk now.